packages/enzyme-adapter-react-16/src/ReactSixteenAdapter.js

```diff
@@ -256,8 +276,9 @@ function toTree(vnode) {
         rendered: childrenToTree(node.child),
       };
     }
-    case FiberTags.Lazy:
+    case FiberTags.Lazy: {
```

Comment:
the curlies are only needed when there's block-scoped variables inside the case; let's revert this.

Reply:
I reverted it!